tsconfig.json

```diff
@@ -1,9 +1,10 @@
 {
   "compilerOptions": {
-    "target": "es2016",
+    "target": "es2021",
```

Comment:
Change is necessary to use `.replaceAll`. It does not seem to have broken anything, and still builds okay.